pandas/core/base.py

```diff
@@ -989,6 +1013,12 @@ def is_monotonic_decreasing(self) -> bool:
         Returns
         -------
         bool
+
+        Examples
+        --------
+        >>> s = pd.Series([3, 2, 2, 1])
+        >>> s.is_monotonic_decreasing
```

Comment:
for this and the above, could we add an example for when the return value is `False`?

In general, for functions which return `-> bool`, it's probably good to have a simple example of both cases (something quick and simple is fine)

Reply:
Done, thanks for the instructions.